src/Altinn.App.Core/Internal/Process/ProcessEventHandlingDelegator.cs

```diff
@@ -72,6 +72,7 @@ IEnumerable<IProcessTask> processTasks
                         case InstanceEventType.process_StartEvent:
                             break;
                         case InstanceEventType.process_StartTask:
+                            // ! TODO: figure out why taskId can be null here
```

Comment:
It's because some of the events don't have taskIds (StartEvent and EndEvent). Code can be restructured to avoid the ! though.

Reply:
Ahh I see. What would be the change here? Something we can do now (non-breaking) ? I'm happy to try, or I could make a better comment here if it's something we should do later

Reply:
I can give it a shot and commit here.